bidwire/scrapers/massgov/url_scraper_dict.py

```diff
@@ -0,0 +1,42 @@
+UL_CATEGORY_LI = '//ul[@class="category"]/li'
+H2_A_TITLELINK = './h2/a[@class="titlelink"]'
+SPAN_A_TITLELINK = './span/a[@class="titlelink"]'
+DIV_BODYFIELD_P = '//div[contains(@class,"bodyfield")]/p'
+
+CATEGORY_H2_XPATH = [ UL_CATEGORY_LI, H2_A_TITLELINK ]
+BODYFIELD_SPAN_XPATH = [ DIV_BODYFIELD_P, SPAN_A_TITLELINK ]
+
+"""Mapping of relative URL (for EOPSS pages) to the xpath needed 
+    to extract documents (1st xpath for section, 2nd xpath for document link)
+"""
+def get_dict():
```

Comment:
This looks like a constant, it doesn't need to be a method, i.e you can just do something like:
```python
MASSGOV_DICT = {
   [....]
}
```

And then you can access `MASSGOV_DICT` (or a better name 😉 ) from other libraries in the usual way.